purchase_analytic/tests/test_purchase_analytic.py

```diff
@@ -17,65 +18,74 @@ def setUp(self):
             {"name": "manual", "plan_id": analytic_plan.id}
         )
         self.analytic_distribution_manual = {str(analytic_account_manual.id): 100}
+        self.analytic_distribution_manual2 = {str(analytic_account_manual.id): 50}
 
-    def test_analytic_distribution(self):
+    def add_new_order_line(self):
+        return [
+            Command.create(
+                {
+                    "name": self.product_id.name,
+                    "product_id": self.product_id.id,
+                    "product_qty": 1.0,
+                    "product_uom": self.uom_id.id,
```

Comment:
I'd suggest making these params configurable and provide default values, so it may be called without params but also being able to provide some of them.